list/List.js

```diff
@@ -227,9 +225,14 @@ define([
 		 * @private
 		 */
 
-		render: function () {
+		postRender: function () {
```

Comment:
inside _propagateRole, there is a `this.selectionMode = "single";`, and then inside refreshRendering there is a bunch of things that need to be triggered once this prop has changed. 
In order for selectionMode to be part of the next refresh cycle it needs to be outside render.
@wkeese am I doing something wrong?